text/0048-rename-max-mdb-size-var.md

```diff
@@ -22,9 +22,20 @@ N/A
 
 ### IV. Explanation
 
- - Rename `--max-mdb-size` to `--max-index-size`
- - Rename `MAX_MDB_SIZE` to `MAX_INDEX_SIZE`
-
+- Rename `--max-mdb-size` to `--max-index-size`
+- Rename `MAX_MDB_SIZE` to `MAX_INDEX_SIZE`
+- Error handling at index creation should be iso with version 0.20.
+
+When I test the current version of meilisearch in 0.21, the http call on `POST /indexes/:index_uid/documents` is successful and an empty index is created whereas 0.20 returns an error on the API side and the index is not created.
```

Comment:
Not implemented yet, see https://github.com/meilisearch/transplant/issues/204 and https://github.com/meilisearch/transplant/issues/17

The behavior will be the same in v0.21.0

Reply:
Good point! Thanks!